package.json

```diff
@@ -32,13 +32,13 @@
   "devDependencies": {
     "cheerio": "^0.22.0",
     "css-loader": "^0.25.0",
-    "extract-text-webpack-plugin": "^1.0.1",
-    "html-webpack-plugin": "^2.16.0",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
+    "html-webpack-plugin": "^4.0.0-beta.4",
```

Comment:
Why beta versions? Can we use latest stable?

Reply:
`html-webpack-plugin` v4 is still in beta and also `extract-text-webpack-plugin `compatible with html-webpack-plugin v4 is still in beta (the latest stable version of `extract-text-webpack-plugin` is not compatible with `html-webpack-plugin` v4). 

Although html-webpack-plugin v4 is in beta, it is required in the production version of widely used `react-scripts` (see https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/package.json), so please consider merging this PR into the beta version of `html-webpack-inline-source-plugin`.